app/src/Navigators/utils.ts

```diff
@@ -44,6 +44,11 @@ export function navigate(name: keyof RootStackParamList, params: any) {
   }
 }
 
+/**
+ * Reset the navigation state to the provided state.
+ * @param {routes} routes of the navigation.
+ * @param {index} index of the route to reset.
```

Comment:
I think what's inside the curly brackets on the two lines above should be the variable type (`array` or `number`) and then the name of the variables (`routes`, `index`) should come after the closing curly bracket